wagtail_wordpress_import/block_builder.py

```diff
@@ -84,14 +99,16 @@ def build(self):
             counter += 1
             # the builder function for the element tag from config
             builder_function = self.get_builder_function(element)
-
             if builder_function:  # build a block
                 if cached_fallback_value:
                     cached_fallback_value = cached_fallback_function(
                         cached_fallback_value,
                         self.blocks,
                     )  # before building a block write fall back cache to a block
-                self.blocks.append(builder_function(element))  # write the new block
+                if builder_function[0] == "html":
+                    self.blocks.append(builder_function[1](element))
+                elif builder_function[0] == "custom":
+                    self.blocks.append(builder_function[1]().construct_block(element))
```

Comment:
**Comment**: This isn't terribly clear, though I see where it's come from. Do we need to store these as "html" and "custom"? Can we use something more readable, maybe an `isinstance` check instead, then we wouldn't need the tuple indexing?

Reply:
I will look again at this.